### PR TITLE
Trails

### PR DESCRIPTION
**Pull request by niicode**
This was a form to test my basic html and css skills during the microverse trails

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                <input type="email" class="input form-control" placeholder="Email Address" aria-label="Email Address">
+                <input type="number" class="input form-control" placeholder="Age(optional)" aria-label="Age">
+                <select class=" input form-select" aria-label="Default select example">
+                    <option selected>What best describes your role</option>
+                    <option value="1">Student</option>
+                    <option value="2">Teacher</option>
+                    <option value="3">Software Engineer</option>
+                    <option value="3">Cloud Engineer</option>
+                </select>
+                <div class="input form-control">
```
Comment (niicode) [inline index.html]: I think it would be good if you used html semantic tags like sections as child elements to a Parent div tag.

**Review by niicode — COMMENTED** [review]
@niicode great job done just that there are a few changes and we are good to go🕺
✔︎ No conflicts
✔︎ No linter errors
✔︎ Good Commit messages

**Comment by niicode** [comment]
@niicode great job done there is nothing else to do than merge🎉
✔︎ No conflicts
✔︎ No linter errors
✔︎ Good Commit messages

**Review by Ugboaja-Uchechi — COMMENTED** [review]
Hi @niicode , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has already been merged. Please, make sure you always submit an open pull request. Please open a pull request before submitting it for review. 🙏 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Pull request merged**